src/main/webapp/app/shared/components/not-released-tag.component.ts

```diff
@@ -9,6 +9,6 @@ import { Exercise } from 'app/entities/exercise.model';
 })
 export class NotReleasedTagComponent {
     @Input() public exercise: Exercise;
-    @Input() public noMargin?: boolean;
+    @Input() public noMargin?: boolean = true;
```

Comment:
```suggestion
    @Input() public noMargin: boolean = true;
```
Do we have an opinion about keeping/omitting the type as well if we have a default value? I'm open for both

Reply:
I know that when initializing booleans, the type is often omitted because it is trivial. However, in Artemis, we have always included it so far. We can discuss this in the developer meeting and consider removing it in the future. We should definitely keep other type annotations.